### PR TITLE
Added MISA reference TIP

### PR DESCRIPTION
**Pull request by ilantz**
I've added a TIP to view the MISA catalog to view our DMRAC offerings, and changed the valimail mention to be more inclusive for the MISA offerings we have.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ description: "Learn how to configure Domain-based Message Authentication, Report
 # Use DMARC to validate email in Office 365
 
 Domain-based Message Authentication, Reporting, and Conformance ([DMARC](https://dmarc.org)) works with Sender Policy Framework (SPF) and DomainKeys Identified Mail (DKIM) to authenticate mail senders and ensure that destination email systems trust messages sent from your domain. Implementing DMARC with SPF and DKIM provides additional protection against spoofing and phishing email. DMARC helps receiving mail systems determine what to do with messages sent from your domain that fail SPF or DKIM checks.
-  
+
+> [!TIP]
+> Visit the [Microsoft Intelligent Security Association (MISA)](https://www.microsoft.com/misapartnercatalog) catalog to view 3rd party vendors offering DMARC reporting for Office 365.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md]:
```suggestion
> Visit the [Microsoft Intelligent Security Association (MISA)](https://www.microsoft.com/misapartnercatalog) catalog to view third-party vendors offering DMARC reporting for Office 365. 
```

File: microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md [hunk 2]
```diff
@@ -74,7 +77,7 @@ Microsoft's DMARC TXT record looks something like this:
 _dmarc.microsoft.com.   3600    IN      TXT     "v=DMARC1; p=none; pct=100; rua=mailto:d@rua.agari.com; ruf=mailto:d@ruf.agari.com; fo=1" 
 ```
 
-Microsoft sends its DMARC reports to [Agari](https://agari.com), a 3rd party. Agari collects and analyzes DMARC reports. Additionally, Valimail offers [free DMARC monitoring services for Office 365](https://go.valimail.com/microsoft.html).
+Microsoft sends its DMARC reports to [Agari](https://agari.com), a 3rd party. Agari collects and analyzes DMARC reports. Please visit the [MISA catalog](https://www.microsoft.com/misapartnercatalog) to view more 3rd party vendors offering DMARC reporting for Office 365.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md]:
```suggestion
Microsoft sends its DMARC reports to [Agari](https://agari.com), a third party. Agari collects and analyzes DMARC reports. Please visit the [MISA catalog](https://www.microsoft.com/misapartnercatalog) to view more third-party vendors offering DMARC reporting for Office 365.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - Please spell out ordinal numbers (Microsoft Style Guide).

**Comment by mypil** [comment]
@ilantz - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by ilantz** [comment]
Thx for the copy/edit Johan

**Comment by mypil** [comment]
@ilantz - it appears that the copy/edit suggestions made by @JohanFreelancer9  was not merged so those changes are still not integrated to this PR you created. Can you please make the suggested changes so we can proceed to the next phase? Thank you.

**Review by ilantz — COMMENTED** [review]
3rd > third

**Comment by ilantz** [comment]
think i got it now :)

**Comment by mypil** [comment]
@ilantz - thanks for merging the copy/edit suggestions.

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. 


Thank you.

cc: @kenwith

**Pull request merged**